alphavilleseries/alphaville_series_service_test.go

```diff
@@ -0,0 +1,154 @@
+package alphavilleseries
+
+import (
+	"os"
+	"testing"
+
+	"github.com/Financial-Times/base-ft-rw-app-go/baseftrwapp"
+	"github.com/Financial-Times/neo-utils-go/neoutils"
+	"github.com/jmcvetta/neoism"
+	"github.com/stretchr/testify/assert"
+)
+
+var alphavilleSeriesDriver baseftrwapp.Service
+
+func TestDelete(t *testing.T) {
+	assert := assert.New(t)
+	uuid := "12345"
+
+	alphavilleSeriesDriver = getAlphavilleSeriesCypherDriver(t)
+
+	alphavilleSeriesToDelete := AlphavilleSeries{UUID: uuid, PrefLabel: "Test", TmeIdentifier: "TME_ID"}
+
+	assert.NoError(alphavilleSeriesDriver.Write(alphavilleSeriesToDelete), "Failed to write alphavilleSeries")
+
+	found, err := alphavilleSeriesDriver.Delete(uuid)
+	assert.True(found, "Didn't manage to delete alphavilleSeries for uuid %", uuid)
```

Comment:
Ah yes - copied from the sections r/w... Updated now both here and in the bottom test.